packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts

```diff
@@ -316,6 +316,10 @@ export interface SSIPrimaryButtonConfig extends ComponentConfig {
         mainContainer: {
             backgroundColor: string
         }
+        button: {
+            width?: string | number
```

Comment:
why do we allow strings here? this should really be a number? same for height

Reply:
if the argument will be to set units, then i think these are only used for styles where react is handling the units for you

Reply:
"10 rem" or something? %?

Reply:
ye i guess % will be hard then. i guess it's fine then. personally i hardly use them so i always forget it exists.